src/services/api.js

```diff
@@ -840,3 +840,19 @@ SDKWrapper.prototype.fetchSendWyreOffers = function (fromAsset: string, toAsset:
     })
     .catch(() => ({ error: true }));
 };
+
+SDKWrapper.prototype.getUserWallets = function (accessToken) {
```

Comment:
```suggestion
SDKWrapper.prototype.getUserWallets = function (accessToken: string) {
```


Not sure if we could also add the return type, like `Promise<Account[]>`

Reply:
I'll check and update typing